### DISCUSSION TITLE
I have a question for ChanLink VRF

### DISCUSSION THREAD

**Discussion by kaizendev2015** [samurai-techlead/Coinflip-Contract-Simple-VRF#2] [Q&A]
Why you used the chain link vrf to generate Randomess

**Comment by samurai-techlead** [comment]
Thanks for  your interest for my repository.
As you know there are some ways to generate the verified random number by interacting with off-chain.
I think mostly developers are using the Oracle Provable API and Chainlink VRF API.
We can use anything.
But in the Ethereum Layer2 Solution-Polygon Network, Oracle Provable API is not working well, but the ChainLink Provable API is suitable for Polygon Network.
So, I used the ChainLink VRF.
I hope it will be helpful for you.
